presentation/src/main/java/me/androidbox/presentation/navigation/NavigationGraph.kt

```diff
@@ -193,7 +221,52 @@ fun NavigationGraph(
                 },
             )
         }
-        
+
+        /* Task Detail Screen */
+        composable(
+            route = Screen.TaskDetailScreen.route,
+            arguments = listOf(
+                navArgument(Screen.TaskDetailScreen.TASK_ID) {
+                    type = NavType.StringType
+                    nullable = true
+                    defaultValue = null
+                }, navArgument(Screen.TaskDetailScreen.MENU_ACTION_TYPE) {
+                    type = NavType.StringType
+                    defaultValue = AgendaMenuActionType.OPEN.name
+                }),
+            deepLinks = listOf(navDeepLink {
+                uriPattern = AgendaDeepLinks.TASK_DEEPLINK
+            })
+        ) {
+            val taskDetailViewModel: TaskDetailViewModel = hiltViewModel()
+            val taskDetailScreenState by taskDetailViewModel.taskDetailScreenState.collectAsStateWithLifecycle()
+            val title = it.savedStateHandle.get<String>(ContentType.TITLE.name) ?: "New Event"
```

Comment:
Here it should be "New Task" instead of New Event

But i wouldnt add default params here tbh
If theres no string, then just pass it as null. And have the state set the default values